pkg/kubelet/types/types_test.go

```diff
@@ -23,6 +23,99 @@ import (
 	"k8s.io/kubernetes/pkg/api/v1"
 )
 
+func TestConvertToTimestamp(t *testing.T) {
+	var cases = []struct {
+		timestamp string
+	}{
+		{
+			timestamp: "2017-02-17T15:34:49.830882016+08:00",
+		},
+	}
+	for _, data := range cases {
+		actual := ConvertToTimestamp(data.timestamp).GetString()
+		if actual != data.timestamp {
+			t.Errorf(
+				"failed ConvertToTimestamp:\n\texpected: %v\n\t  actual: %v",
+				data.timestamp,
+				actual,
+			)
+		}
+	}
+}
+
+func TestLen(t *testing.T) {
```

Comment:
How about keep three functions here, this can make the test logic simple and also the function name easy to understand.

```
func TestLen(t *testing.T)
func TestSwap(t *testing.T)
func TestLess(t *testing.T)
```